front/app/components/ParticipationCTABars/EmbeddedSurveyCTABar/index.tsx

```diff
@@ -48,24 +47,24 @@ export const EmbeddedSurveyCTABar = ({ phases, project }: CTABarProps) => {
     }
   }, [divId]);
 
-  const scrollTo = useCallback(
-    (id: string, shouldSelectCurrentPhase = true) =>
-      (event: FormEvent) => {
-        event.preventDefault();
-        const isOnProjectPage = pathname.endsWith(
-          `/projects/${project.attributes.slug}`
-        );
-
-        currentPhase && shouldSelectCurrentPhase && selectPhase(currentPhase);
-
-        if (isOnProjectPage) {
-          scrollToElement({ id, shouldFocus: true });
-        } else {
-          clHistory.push(`/projects/${project.attributes.slug}#${id}`);
-        }
-      },
-    [currentPhase, project, pathname]
-  );
+  const scrollToSurvey = useCallback(() => {
```

Comment:
Noticed that we've removed the `(event: FormEvent) => {event.preventDefault();` here - was that intentional?